### PR TITLE
[Chore] Expendad test coverage of create-test-run-modal

### PR DESCRIPTION
**Pull request by stefanwaldhauser**
### Checklist
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).
- [x] Client: I added multiple integration tests (Jest) related to the features (with a high test coverage)
 

Expanded the test coverage of test-run-modal for exams: 94.12%

### CODE REVIEW

**Review by derLalla — DISMISSED** [review]
good improvement of the underlying expect() method.

**Review by FuchsDominik — DISMISSED** [review]
Looks good, nice test :)

**Review by anditurdiu — APPROVED** [review]

File: src/test/javascript/spec/component/exam/test-run/create-test-run-modal.component.spec.ts [hunk 1]
```diff
@@ -1,51 +1,88 @@
...
+            expect(testRun.workingTime).to.equal(1800);
+        });
+    });
+
     describe('Ignore Exercise groups', () => {
         it('should ignore exercise groups with no exercises', function () {
             comp.exam = exam;
             comp.exam.exerciseGroups = [exerciseGroup1, exerciseGroup2];
             fixture.detectChanges();
-            expect(comp.exam.exerciseGroups!.length).toBe(1);
```
Comment (FrankeLukas) [inline src/test/javascript/spec/component/exam/test-run/create-test-run-modal.component.spec.ts:48]: This test doesn't do anything anymore if I see this correctly? (No except statement.)

**Review by FrankeLukas — CHANGES_REQUESTED** [review]

File: src/test/javascript/spec/component/exam/test-run/create-test-run-modal.component.spec.ts (hunk 1 above)
Comment (stefanwaldhauser) [reply src/test/javascript/spec/component/exam/test-run/create-test-run-modal.component.spec.ts:48]: good catch added it back in :) done in b69401a5aabd8b074c285e88405213360de6465b

**Review by krusche — APPROVED** [review]

**Pull request merged**